### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by akib-raihan-sazim**
**Description of changes**
- Added login page
- Hardcoded authentication

**Associated ticket link**
https://trello.com/c/9a1ALcE7

**Screenshot / Screen Recording**

https://github.com/user-attachments/assets/773bd524-d891-49e5-aefb-2507e3ff9ead

### CODE REVIEW

File: pages/LoginPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import LoginPageContainer from '@/modules/LoginPage/containers/LoginPageContainer'
+import React from 'react'
```
Comment (ishmam-iqbal-sazim) [inline pages/LoginPage/index.tsx:2]:
```suggestion
import LoginPageContainer from "@/modules/LoginPage/containers/LoginPageContainer"
import React from "react"
```

File: pages/LoginPage/index.tsx
Comment (ragib-hossain-sazim) [inline pages/LoginPage/index.tsx:1]: Same comment no `Page` postfix required in pages folder. `LoginPage` => `Login`

Comment (ragib-hossain-sazim) [inline pages/LoginPage/index.tsx:1]: `index.tsx` => `index.page.tsx`

File: modules/LoginPage/containers/LoginPageContainer.tsx
Comment (ragib-hossain-sazim) [inline modules/LoginPage/containers/LoginPageContainer.tsx:1]: `LoginPageContainer` => `LoginContainer`

File: modules/LoginPage/containers/LoginPageContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  return (
+    <div className="loginWrapper">
+      <div className="stars"></div>
+      <div className="twinkling"></div>
+      <div className="clouds"></div>
+      <Container size={420} my={40}>
+        <div className="feature-list">
+          <Title className="title-login">Welcome back!</Title>
+          <Paper withBorder shadow="md" p={30} mt={30} radius="md">
+            <form onSubmit={form.onSubmit(handleSubmit)}>
```
Comment (ragib-hossain-sazim) [inline modules/LoginPage/containers/LoginPageContainer.tsx:54]: Create a seperate form component for LoginForm and move form related stuff there. Container should be a collection of multiple components.

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**